Week3/homework/3-step3.js

```diff
@@ -3,23 +3,42 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
-  return str;
+  let newStr = '';
+  if (num >= 1) {
+    for (let i = 0; i < num; i++) {
```

Comment:
As the loop body will not execute when `num` is zero or less, there is no need to test for this separately with the `if` statement of line 7.